test/e2e/provisioning/helper.go

```diff
@@ -33,9 +33,10 @@ var (
 	scenarios = buildScenarios()
 
 	versions = []*semver.Version{
-		semver.MustParse("v1.15.6"),
-		semver.MustParse("v1.16.3"),
-		semver.MustParse("v1.17.0"),
+		semver.MustParse("v1.16.15"),
```

Comment:
Nit: similar to 1.15, we should also drop 1.16 at some point, since we don't support it in KKP/K1 anymore.